catalogs/catalog-kafka/src/test/java/com/datastrato/gravitino/catalog/kafka/integration/test/CatalogKafkaIT.java

```diff
@@ -0,0 +1,388 @@
+import com.datastrato.gravitino.rel.SchemaChange;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.Config;
+import org.apache.kafka.clients.admin.TopicDescription;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.common.config.ConfigResource;
+import org.apache.kafka.common.config.TopicConfig;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+
+@Tag("gravitino-docker-it")
+public class CatalogKafkaIT extends AbstractIT {
+  private static final Logger LOG = LoggerFactory.getLogger(CatalogKafkaIT.class);
+  private static final ContainerSuite CONTAINER_SUITE = ContainerSuite.getInstance();
+  private static final String METALAKE_NAME =
+      GravitinoITUtils.genRandomName("catalogKafkaIT_metalake");
+  private static final String CATALOG_NAME =
+      GravitinoITUtils.genRandomName("catalogKafkaIT_catalog");
+  private static final String DEFAULT_SCHEMA_NAME = "default";
+  private static final String PROVIDER = "kafka";
+  private static final String KAFKA_LOGS_DIR = "/opt/kafka/logs";
+  private static GravitinoMetalake metalake;
+  private static Catalog catalog;
+
+  private static String kafka_bootstrap_servers;
+  private static AdminClient adminClient;
+
+  @BeforeAll
+  public static void startup() {
+    CONTAINER_SUITE.startKafkaContainer();
+    kafka_bootstrap_servers =
+        String.format(
+            "%s:%d",
+            CONTAINER_SUITE.getKafkaContainer().getContainerIpAddress(), DEFAULT_BROKER_PORT);
+    adminClient = AdminClient.create(ImmutableMap.of(BOOTSTRAP_SERVERS, kafka_bootstrap_servers));
+
+    createMetalake();
+    createCatalog();
+  }
+
+  @AfterAll
+  public static void shutdown() {
+    client.dropMetalake(NameIdentifier.of(METALAKE_NAME));
+    if (adminClient != null) {
+      adminClient.close();
+    }
+
+    copyKafkaLogs();
```

Comment:
So if we put the Kafka log here, does it mean that for each IT, we have to call this method, otherwise the log will be missing, can we move to the container related code?